README.md

```diff
@@ -1,2 +1,18 @@
 lightly modified from https://github.com/mitsuhiko/flask-sqlalchemy/tree/master/examples
 
+## Dev setup
+
+    $ export FLASK_APP=todo.py
+    $ export TODO_SETTINGS=/path/to/local/config.cfg
```

Comment:
should this be `todo_local.cfg` not `config.cfg` or does it not matter because they need to change it anyway

Reply:
Yea, figured it didn't really matter